lib/teaspoon/engine.rb

```diff
@@ -101,7 +101,7 @@ def render_exception(_env, exception)
 
 begin
   require 'action_view'
-  if ActionView::VERSION::STRING >= '4.2.5'
+  if ActionView.gem_version >= Gem::Version.new('4.2.5')
```

Comment:
One question I had while working on this was if this patch should continue to be applied on Action View versions 5.0+. Even if this patch doesn't get applied in Rails 5.0+, things appear to continue to work.

Reply:
Scratch that, `'5.0.0' >= '4.2.5'`, so this patch is applied on 5.0+!